test/sai_test/config/route_configer.py

```diff
@@ -739,6 +749,31 @@ def create_nexthop_by_rif(self, rif, nexthop_device: Device):
 
         return nhopv4, nhopv6
 
+    def create_nexthop_group_by_nexthops(self, dest_device: Device):
+        """
+        Create nexthop group by nexthops.
+        """
+        self.test_obj.nhop_group1 = sai_thrift_create_next_hop_group(
+            self.test_obj.client, type=SAI_NEXT_HOP_GROUP_TYPE_ECMP)
+
+        self.test_obj.nh_group_member1 = sai_thrift_create_next_hop_group_member(
```

Comment:
we define two kind of data objects, device and dut, for persist then in a file then we can load them.
Why add those attribute in test_obj?